README.md

```diff
@@ -26,3 +25,7 @@ Basic express server using modularization, middleware and tests.
 
 - created with AWW - A Web Whiteboard
 - [UML](UML.jpg)
+
+## DynamoDB
+
+- [dynamoDB](https://us-west-2.console.aws.amazon.com/dynamodb/home?region=us-west-2#tables:selected=people;tab=items)
```

Comment:
Something that I would like to see added are what inputs are required for each route and what the expected output is!

Reply:
If it's on your UML the problem is that image is cropped so I can't see everything